open-sphere-base/core/src/main/java/io/opensphere/core/util/Colors.java

```diff
@@ -34,17 +34,17 @@
     /** Other edging (tabs, tables), disabled text (128, 128, 128). */
     public static final Color LF_SECONDARY2 = Color.GRAY;
 
-    /** Background, menu bar (83, 83, 102). */
-    public static final Color LF_SECONDARY3 = Color.decode("#535366");
+    /** Background, menu bar (83, 83, 83). */
+    public static final Color LF_SECONDARY3 = Color.decode("#535353");
```

Comment:
Is the intention in general to match MIST Web's colors?  Note there are some colors defined at the top here (OPENSPHERE_*) that are intended to match Web's colors, although they may be out of date.

Reply:
Indeed... this is actually for a secondary ticket that probably should have been on its own branch, to ditch the purple in favor of grayscale.